visualize_training_set.py

```diff
@@ -0,0 +1,68 @@
+import os
+import tensorflow as tf
+import sys
+import cv2
+import numpy as np
+
+class_names = ['car', 'watcher', 'base', 'armor_red', 'armor_blue']
+
+IMAGE_FEATURE_MAP = {
+    'image/encoded': tf.io.FixedLenFeature([], tf.string),
+    'image/object/bbox/xmin': tf.io.VarLenFeature(tf.float32),
+    'image/object/bbox/ymin': tf.io.VarLenFeature(tf.float32),
+    'image/object/bbox/xmax': tf.io.VarLenFeature(tf.float32),
+    'image/object/bbox/ymax': tf.io.VarLenFeature(tf.float32),
```

Comment:
```suggestion
    'bbox': tf.io.FixedLenFeature([], tf.string),
```

Later when you parse_example, you can treat `image_features['bbox']` as a (N, 4) tensor.